CHANGELOG.md

```diff
@@ -1,3 +1,28 @@
+# 0.34.0-alpha
+* Update dependencies
+* Add weekly dependabot updates (jylitalo)
+* Add support for regional sts endpoints (matt-mercer)
```

Comment:
👀 
@cristiangreco I don't remember exactly what was the problem we had trying regional STS endpoints, but the blocker was before hitting the exporter, right?

Reply:
I think our problem was around "job validation". Anyway chatted with Kyle yesterday about this specific change and it shouldn't impact us as we use a custom `session`.

Reply:
Thanks! 🙌